cypress/tsconfig.json

```diff
@@ -1,18 +1,13 @@
 {
+  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "esModuleInterop": true,
+    "composite": false,
     "incremental": true,
-    "lib": ["DOM", "DOM.Iterable", "ESNext"],
-    "noEmit": true,
-    "strict": true,
```

Comment:
yay no strict for tests!

Reply:
Erm i think it is strict, just inherited. 

Tbh I'm not sure what this does, but if you're against it you should pr it to explicitly not be strict with a comment so it doesn't backslide.